docs/reference/project/functions.mdx

```diff
@@ -1 +1,30 @@
-TODO...
+A Nitric function is described by its definition in a `nitric.yaml` file. Nitric functions are a type of [Service](../../intro/services) that offers a portable way to develop function-as-a-service (FaaS) applications, which can then be deployed to
+popular Serverless offerings like AWS Lamda, Google CloudRun and Azure AppService.
+
+## Properties
+
+`name` the name of the Nitric function. Must be unique to the stack file as it is used as the reference to the function for entrypoints and apis.
+
+`handler` the path to the function handler, relative to the root of the project.
+
+There are optional properties for when triggering a function with a topic.
+
+`triggers` the [topic(s)](./topics.mdx) that this function subscribes to.
```

Comment:
For other reference docs I've used a table to describe properties. Nothing wrong with this format, but we should pick one to maintain consistency.

Reply:
I will change it to the table